force-app/main/default/objects/Journey__c/fields/Manager_Email__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Manager_Email__c</fullName>
+    <caseSensitive>true</caseSensitive>
+    <description>Stores the email of the manager that is assigned to each Mainlydigital candidate.</description>
+    <externalId>false</externalId>
+    <label>Manager Email</label>
+    <required>true</required>
+    <trackHistory>false</trackHistory>
+    <trackTrending>false</trackTrending>
+    <type>Email</type>
+    <unique>true</unique>
```

Comment:
Because the `Manager Email` field is marked as unique, this means that each manager would need a different email address to manage more than one candidate at a time.